stratos-service/admin-ui/src/app.css

```
@@ -126,22 +126,29 @@
-    background 0.25s ease,
-    color 0.25s ease;
-}
+/*
+ * Element defaults belong in the base layer: unlayered rules outrank every
+ * Tailwind layer, so an unlayered `h1, h2` here would beat `font-sans` and
+ * `not-italic` utilities at the call site no matter how specific they are.
+ */
+@layer base {
+  body {
+    @apply bg-background font-sans text-foreground antialiased;
+    margin: 0;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)